ddsrouter_yaml/test/unittest/configuration/YamlReaderConfigurationTest.cpp

```diff
@@ -149,20 +147,17 @@ TEST(YamlReaderConfigurationTest, get_ddsrouter_configuration_no_version)
                 R"(
             participants:
               - name: "P1"
-                kind: "void"
+                kind: "echo"
               - name: "P2"
-                kind: void"
+                kind: echo"
             )";
 
         Yaml yml = YAML::Load(yml_configuration);
 
-        // Load configuration
-        ddsrouter::core::DdsRouterConfiguration configuration_result =
-                ddsrouter::yaml::YamlReaderConfiguration::load_ddsrouter_configuration(yml);
-
-        // Check is not valid
-        utils::Formatter error_msg;
-        ASSERT_FALSE(configuration_result.is_valid(error_msg)) << error_msg;
+        // Load configuration and check is not valid
+        ASSERT_THROW(
+            ddsrouter::yaml::YamlReaderConfiguration::load_ddsrouter_configuration(yml),
```

Comment:
1. This does not test what it says it tests. It is testing that setting a wrong yaml (`"` missing) does fail, but it is not the purpose of this test
2. please, add negative cases under `_negative` tests, so if a logError is printed, we now it is expected